### ISSUE TITLE
SkyUI icons for items. 

### ISSUE THREAD

**Issue by NimRo97** [Ryan-rsm-McKenzie/QuickLootRE#67]
Would it be possible to implement (optional) support for item icons in the quick loot menu?

**Comment by SpikerzZ406** [comment]
I agree, that would be awesome

**Comment by MartinHaeusler** [comment]
I second this. The quick loot menu without icons is nearly unusable because yoh have to read all the item names, whereas icons allow for quick identification. Also: please left-align item names, centering is bad for quick reading...

**Comment by zauberparacelsus** [comment]
I also agree with this.  I'd also like to see price and weight listed alongside every item, rather than listed at the bottom for only the currently-highlighted item.

**Comment by InitialDet** [comment]
This! Please

**Comment by calebjosephchambers** [comment]
Also Let The User Edit The Position Of Menu.

**Comment by MartinHaeusler** [comment]
Any news on this one? Being able to quickly tell the item types apart is super important for fluent gameplay...

**Comment by CrazieIvan** [comment]
The lack of colored icons (or other type of categorization) is the only reason I find it difficult to use this mod.  I'm a slow reader, and since all the item text looks the same, I have to spend too much time reading each one and deciding what I need.  
Thank you for your consideration!

**Comment by MartinHaeusler** [comment]
Leaving a bump here, as this is really important.

**Comment by derickso** [comment]
Another user has added this functionality

https://www.nexusmods.com/skyrimspecialedition/mods/69980
